man/tpm2_nvwrite.1.md

```diff
@@ -40,6 +40,11 @@ If _FILE_ is not specified, it defaults to stdin.
     should follow the authorization formatting standards, see section
     "Authorization Formatting".
 
+  * **-X**, **--hmac-auth**
```

Comment:
We're not doing this, delete this.

Reply:
Fixed. Updated the authorization.md file instead to indicate hmac auth mechanism.